metricflow/test/snapshots/test_dataflow_to_sql_plan.py/SqlQueryPlan/BigQuerySqlClient/test_join_to_time_spine_node_with_offset_to_grain__plan0_optimized.sql

```diff
@@ -0,0 +1,39 @@
+-- Join to Time Spine Dataset
+SELECT
+  subq_11.metric_time AS metric_time
```

Comment:
Was this supposed to be truncated to month as well?

Reply:
Nope! We want to keep that at daily so each day shows the value at the start of the grain. (Again, not very useful for an individual metric, but useful when comparing to another metric.)